app/models/admin_shelters.rb

```diff
@@ -0,0 +1,4 @@
+class AdminShelters < ApplicationRecord
```

Comment:
No need for this model either. You can grab applications from shelters by adding the extra association that you have in your shelter model that builds a shortcut from Shelters to Applications